crates/bevy_window/src/window.rs

```diff
@@ -283,6 +292,24 @@ impl Window {
     pub fn set_physical_cursor_position(&mut self, position: Option<DVec2>) {
         self.internal.physical_cursor_position = position;
     }
+
+    /// The window's current theme
+    ///
+    /// ## Platform-specific
+    ///
+    /// - iOS / Android / Web / x11: Unsupported.
+    pub fn window_theme(&self) -> Option<WindowTheme> {
+        self.internal.window_theme
+    }
+
+    /// Set the window's theme
+    ///
+    /// ## Platform-specific
+    ///
+    /// - iOS / Android / Web / x11: Unsupported.
```

Comment:
Note that _setting_ the theme is supported on x11 by winit. The doc string is innacurate (see rust-windowing/winit#2814)

```suggestion
    /// Ignored on iOS, Android and Web
```